src/xmpp/presence.c

```diff
@@ -656,6 +658,69 @@ _available_handler(xmpp_stanza_t* const stanza)
 
     if (g_strcmp0(xmpp_presence->jid->barejid, my_jid->barejid) == 0) {
         connection_add_available_resource(resource);
+        char* account_name = session_get_account_name();
+        int max_sessions = accounts_get_max_sessions(account_name);
+        char* cur_resource = accounts_get_resource(account_name);
+        if (max_sessions > 0) {
+            int res_count = connection_count_available_resources();
+            if (res_count > max_sessions && g_strcmp0(cur_resource, resource->name)) {
+                ProfWin* console = wins_get_console();
+                ProfWin* current_window = wins_get_current();
+                auto_gchar gchar* message = g_strdup_printf("Max sessions alarm! (%d/%d devices in use)", res_count, max_sessions);
+                win_println(console, THEME_RED, "|", message);
+                if (console != current_window) {
+                    win_println(current_window, THEME_RED, "|", message);
+                }
+                notify(message, 10000, "Security alert");
```

Comment:
Shouldn't this be enclosed in a `if (prefs_get_boolean(PREF_NOTIFY_SECURITY_ALERTS))`? (`PREF_NOTIFY_SECURITY_ALERTS` does not exist yet, maybe we should choose another name!?)

Reply:
I think on the next PR with /privacy command we can integrate the setting

Reply:
Maybe add it to the tasklist in #1825 ? or open a new tracking issue for the `/privacy` command?

Reply:
i added a comment to the new privacy issue